src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -430,14 +430,14 @@ class Variable
                 Await state.AssertSelectedCompletionItem(displayText:="as", isSoftSelected:=True)
 
                 state.SendTypeChars(", va")
-                Await state.AssertSelectedCompletionItem(displayText:="var", isSoftSelected:=True)
+                Await state.AssertSelectedCompletionItem(displayText:="var", isHardSelected:=True)
```

Comment:
FYI @CyrusNajmabadi Merging Neal's latest parsing changes with the completion tests I had added in master requires those tests to be updated. I think the updated expectations are an improvement.